### PR TITLE
plot jobs set priority 100

### PR DESCRIPTION
**Pull request by ahnitz**
I'm not sure this is the right solution, but it might help with #505.

### CODE REVIEW

**Comment by ahnitz** [comment]
Ok, if I've understood the dagman page documentation, I think this is indeed what we want and should ensure that these jobs are submitted first. 

http://research.cs.wisc.edu/htcondor/manual/v7.6/2_10DAGMan_Applications.html

File: pycbc/workflow/plotting.py [hunk 1]
```diff
@@ -45,6 +45,10 @@ class PlotExecutable(Executable):
     """ plot executable
     """
     current_retention_level = Executable.FINAL_RESULT
+    
+    # plots and final results should get the highest priority
+    # on the job queue
+    self.set_priority(100)
```
Comment (spxiwh) [inline pycbc/workflow/plotting.py]: Change this to 1000.

I think priority seems to be set (automatically) in intervals of 10 according to the depth of a DAX. So a dax with 10 levels might actually start setting priorities to 100.

Comment (ahnitz) [inline pycbc/workflow/plotting.py]: Ok, done.

**Comment by ahnitz** [comment]
Duncan,

Does this need further testing?

**Pull request merged**